### PR TITLE
update icon: gradle (original, original-wordmark)

### PR DESCRIPTION
**Pull request by lunatic-fox**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened

## Features
- Remove text bad positioning
- Remove `plain` and `plain-wordmark` versions
- Add `original` and `original-wordmark` versions
- Alias `original` to `plain` and `original-wordmark` to `plain-wordmark`
- `.editorconfig` is trimming trailing whitespaces

**This PR closes NONE**

## Notes
Sources: [**Gradle website**](https://gradle.org/) and [**Gradle Github**](https://github.com/gradle/gradle)

### CODE REVIEW

**Comment by lunatic-fox** [comment]
On `devicon.json` `gradle` entry modifications goes from [*line 3736 - 3766*](https://github.com/devicons/devicon/pull/1816/files#diff-8fb0089a81b777eae04d6ac8f37becd0a07c41dd808f17e9e90eafb7411a8f60R3736-R3766)

**Comment by lunatic-fox** [comment]
### Screenshots

File: devicon.json [hunk 1]
```diff
@@ -3744,16 +3744,25 @@
         ],
```
Comment (Panquesito7) [inline devicon.json:3744]: Might want to add `open-source` to the tags. 🙂

Comment (lunatic-fox) [reply devicon.json:3745]: Done! 👍🏼

**Review by Panquesito7 — APPROVED** [review]
Awesome. Thanks! 🚀

**Pull request merged**